grafana/grafana
# Explore/Loki: Fix defaulting to instant query when switching from Prometheus

**What this PR does / why we need it**:

When importing query from Prometheus, we want to pass only specific query props, not all as it might result in unexpected results. In this case, when we switched from Prometheus to Loki, the instant query param was `true`, which is incorrect.
This PR fixes it.

More info: https://raintank-corp.slack.com/archives/C9T1FLN9K/p1633695475293500

**To test:**
1. Open Explore
2. Select Prometheus data source
3. Switch to Loki data source - the query type should be range`
